ksql-docker/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2018 Confluent Inc.
+  ~
+  ~ Licensed under the Confluent Community License (the "License"); you may not use
+  ~ this file except in compliance with the License.  You may obtain a copy of the
+  ~ License at
+  ~
+  ~ http://www.confluent.io/confluent-community-license
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>ksql-parent</artifactId>
+    <groupId>io.confluent.ksql</groupId>
+    <version>5.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>ksql-docker</artifactId>
+
+  <properties>
+    <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+    <main-class>io.confluent.ksql.rest.server.KsqlRestApplication</main-class>
```

Comment:
🤔 is this true for both the CLI and the Rest App? Can you test that the image works with our existing docker tutorials, and if not then see what we need to change? I think you asked this question, but is this `main-class` necessary (can we remove it)?

Reply:
Removed the `main-class` and rest. It was needed for enabling executing the created jar via `-jar` which we don't need in the image